<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyCreateDialog</class>
 <widget class="QDialog" name="CompanyCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый поставщик</string>
  </property>
  <widget class="QLineEdit" name="NameInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название организации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя пользователя (поставщика)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateCompanyButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать поставщика</string>
   </property>
  </widget>
  <widget class="QComboBox" name="StorageSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Склад для поставок</string>
   </property>
  </widget>
  <widget class="QComboBox" name="UserSelectInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
